File: Arrays/Easy/1431/1431.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max_candies : int = max(candies)
        output : list = []
        for candy in candies:
            if candy + extraCandies >= max_candies:
                output.append(True)
            else:
                output.append(False)
        return output

File: Arrays/Easy/1431/1431.ts
function kidsWithCandies(candies: number[], extraCandies: number): boolean[] {
  const maxCandies = Math.max(...candies);
  const result: boolean[] = [];

  for (const candy of candies) {
    result.push(candy + extraCandies >= maxCandies);
  }

  return result;
}
